docs/src/componentDocs/Grade/examples/GradeCustomColorExample.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import {ExampleShowcase} from '../../../shared';
+import {Grade} from '@brightlayer-ui/react-native-components';
+
+export const GradeCustomColorExample = (): JSX.Element => (
+  <ExampleShowcase sx={{display: 'flex', justifyContent: 'center'}}>
+    <Grade label="AB" fontColor="#ff0" backgroundColor="#f00" />
```

Comment:
Please move each prop to a separate line for better readability and add dataLine for `fontColor` and `backgroundColor`